scripts/cut_release_branch.py

```diff
@@ -152,6 +154,20 @@ def _execute_branch_cut():
     _verify_target_branch_does_not_already_exist(remote_alias)
     _verify_target_version_is_consistent_with_latest_released_version()
 
+    # The release coordinator should verify that tests are passing on develop
+    # before checking out the release branch.
+    while True:
```

Comment:
why is this in a loop?

The script will stop if the user does not affirm.

Reply:
Whoops, my bad. I amended it to loop only if no answer at all was presented.